STEP/REST.py

```diff
@@ -494,9 +494,16 @@ def list(self,root='ProductsRoot'):
 	#________________________________________________________________________________________________
 	@args.operation(help='get product by id')
 	@args.parameter(name='id', help='the ID of product')
+	@args.parameter(name='inheriteddata', help='Endpoint returns inherited data when parameter is supplied with the value true')
```

Comment:
add the following parameters to the @args.parameter method so that it can be a -i flag to set

short='i', flag=True